testing/bdd/templates/timeBase_clock_clockMode_mandatory.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" ?>
+<tt:tt
+        ebuttp:sequenceIdentifier="testSequence001"
+        ebuttp:sequenceNumber="1"
+        ttp:timeBase="{{ time_base }}"
+{% if clock_mode %}
+  {% if clock_mode == "*?Empty?*" %}
+        ttp:clockMode=""
```

Comment:
Thanks, apologies I always get confused by the empty/missing thing first time I read these. Not sure why!